packages/account/src/Sections/Security/DeactivateAccount/account-has-balance.jsx

```diff
@@ -6,8 +6,13 @@ import { localize, Localize } from '@deriv/translations';
 const getDerivAccount = (client_accounts, login_id) =>
     client_accounts.find((client_account) => client_account.loginid === login_id);
 
-const getMT5Account = (mt5_login_list, login_id) =>
-    mt5_login_list.find((account_obj) => account_obj.login === login_id);
+const getMT5Account = (mt5_login_list, login_id) => {
+    for (const key in mt5_login_list) {
+        if (mt5_login_list[key].login === login_id) {
+            return mt5_login_list[key];
+        }
+    }
+};
```

Comment:
What is the benefit of using classic for loop instead of the `find` method here? it seems the `find` is simpler are more condensed unless there is a special reason.

Reply:
`find` works with arrays, but `mt5_login_list` Is of type `object`. The error was `mt5_login_list.find is not a function`

Reply:
I think it should be array always. if it is not, I guess the bug is something else. 
Also even if being an object is intentional and not a bug, the `mt5_login_list` name is not appropriate for an object the `list` in the name indicates a list.

Reply:
Thank you for the comment. Will look into it!